src/js/index.js

```diff
@@ -1 +1,12 @@
-import '../scss/style.scss';
\ No newline at end of file
+import '../scss/style.scss';
+import Leaderboard from './Leaderboard.js';
+import ScoreForm from './ScoreForm.js';
+
+function init() {
+  const leaderboard = new Leaderboard();
+  const scoreForm = new ScoreForm();
+  leaderboard.setEventListener();
+  scoreForm.setEventListener(leaderboard);
+}
+
```

Comment:
- [ ] in Javascript ES6(which is what we are using now), functions are now defined using `arrow functions` instead of the  `function` keyword. Arrow functions provide a simpler and more concise way to define functions. Kindly change the functions to arrow functions